src/backend/src/services/users.services.ts

```diff
@@ -197,20 +197,31 @@ export default class UsersService {
     const userRole = rankUserRole(user.role);
     const targetUserRole = rankUserRole(targetUser.role);
 
-    if (!isAdmin(user.role)) {
-      throw new AccessDeniedException('Only admins can update user roles!');
+    if (!isHead(user.role)) {
+      throw new AccessDeniedException('Guests, members, and leadership cannot update user roles!');
     }
 
-    if (rankUserRole(role) > userRole) throw new AccessDeniedException('Cannot promote user to a higher role than yourself');
-
     if (targetUserRole >= userRole) {
       throw new AccessDeniedException('Cannot change the role of a user with an equal or higher role than you');
     }
 
-    targetUser = await prisma.user.update({
-      where: { userId: targetUserId },
-      data: { role }
-    });
+    if (user.role === RoleEnum.HEAD) {
```

Comment:
make this one if statement and make the error be heads can only promote to leadership